projects/plugins/jetpack/modules/memberships/class-jetpack-memberships.php

```diff
@@ -480,7 +481,13 @@ public static function user_can_view_post_nocache() {
 		require_once JETPACK__PLUGIN_DIR . 'extensions/blocks/premium-content/_inc/subscription-service/include.php';
 
 		$newsletter_access_level = self::get_newsletter_access_level();
-		$paywall                 = \Automattic\Jetpack\Extensions\Premium_Content\subscription_service();
+
+		if ( null === $newsletter_access_level || Token_Subscription_Service::POST_ACCESS_LEVEL_EVERYBODY === $newsletter_access_level ) {
```

Comment:
I am not confident in `Token_Subscription_Service ` being available here

Reply:
It is included in line 481

```
require_once JETPACK__PLUGIN_DIR . 'extensions/blocks/premium-content/_inc/subscription-service/include.php';
```

Reply:
See this: https://github.com/Automattic/jetpack/blob/94ddae8284758db97529fad30752b9e11a556bb1/projects/plugins/jetpack/extensions/blocks/premium-content/_inc/subscription-service/include.php#L17

Reply:
Seems ok then :)